gunagala/tests/test_psf.py

```diff
@@ -88,15 +88,30 @@ def test_shape(psf):
 
 
 def test_pixellated(psf):
```

Comment:
These tests all need unique names that describe the particular thing that they're testing, e.g. `test_pixellated_square`, `test_pixellated_rectangle`, `test_pixellated_invalid_shape`, etc.